sql_generators/glean_usage/templates/baseline_clients_first_seen_v1.init.sql

```diff
@@ -34,12 +34,13 @@ WITH
   {{ core_clients_first_seen(migration_table) }}
   SELECT
     client_id,
-    submission_date,
-    COALESCE(core.first_seen_date, baseline.first_seen_date) as first_seen_date,
-    sample_id
+    MIN(submission_date) AS submission_date,
+    MIN(COALESCE(core.first_seen_date, baseline.first_seen_date)) AS first_seen_date,
+    sample_id,
   FROM baseline
   LEFT JOIN _core_clients_first_seen core
   USING (client_id)
+  GROUP BY client_id, sample_id
```

Comment:
This should not have a `GROUP BY` clause, because both sides of the join are already unique on `client_id` (and `sample_id` is derived from `client_id`)
```suggestion
    submission_date,
    COALESCE(core.first_seen_date, baseline.first_seen_date) AS first_seen_date,
    sample_id,
  FROM baseline
  LEFT JOIN _core_clients_first_seen AS core
  USING (client_id)
```

If we want to ensure uniqueness here we should add a erroring qualify clause as seen in my other comments